cloudferry/cfglib.py

```diff
@@ -39,6 +39,11 @@
                help='Openstack region name'),
     cfg.StrOpt('service_tenant', default='service',
                help='Tenant name for services'),
+    cfg.StrOpt('nova_endpoint_type', default='publicURL',
```

Comment:
Please name it 'endpoint_type' and use it in other places where we create clients